framework/validators/FileValidator.php

```diff
@@ -258,7 +259,12 @@ protected function validateValue($file)
      */
     public function getSizeLimit()
     {
+        // Max POST size minus some bytes for form data 
+        $post_limit = $this->sizeToBytes(ini_get('post_max_size')) - 1024;
```

Comment:
Method `getSizeLimit()` used only for descriptive purposes, while composing error message.
I suppose it is fine to use plain `ini_get('post_max_size')` for this case.